bundles/org.openhab.ui/web/src/pages/settings/settings-menu.vue

```diff
@@ -81,10 +81,10 @@
               <f7-icon slot="media" f7="calendar" color="gray"></f7-icon>
             </f7-list-item>
           </f7-list>
-          <f7-block-title v-if="addonsLoaded">Add-ons</f7-block-title>
-          <f7-list media-list class="search-list"
-            v-if="$store.getters.apiEndpoint('extensions')"
-          >
+        </f7-col>
+        <f7-col width="100" medium="50" v-if="$store.getters.apiEndpoint('extensions')">
```

Comment:
This messes up the layout when the extensions are present. Could you please revert this back and simply change the v-if condition on f7-block-title to `$store.getters.apiEndpoint('extensions') && addonsLoaded`?

Reply:
I thought it made more sense to break this block? Then you have the three sections in separate blocks (automation and rules, extensions, services). As it is at the moment, you have the "AddOns" section in the same block as the Configuration, but Services is separate.

Doesn't it make more sense to group them separately?

Reply:
Logically, I suppose, but here you have 3 columns of size 50% on medium screens and larger, which means the third gets dropped to a second row and it looks like this:

![image](https://user-images.githubusercontent.com/2004147/85204742-ca207a80-b316-11ea-9b85-56e7486138a3.png)

Which is far less aesthetically pleasing than limiting to 2 columns to get a balanced look:

![image](https://user-images.githubusercontent.com/2004147/85204715-95acbe80-b316-11ea-8b9e-84ef0ae7e945.png)